op-challenger/game/fault/trace/prestates/multi.go

```diff
@@ -9,43 +9,64 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/ethereum-optimism/optimism/op-challenger/game/fault/trace/vm"
 	"github.com/ethereum-optimism/optimism/op-service/ioutil"
 	"github.com/ethereum/go-ethereum/common"
 )
 
 var (
 	ErrPrestateUnavailable = errors.New("prestate unavailable")
+
+	// supportedFileTypes lists, in preferred order, the prestate file types to attempt to download
+	supportedFileTypes = []string{".bin.gz", ".json.gz", ".json"}
```

Comment:
Should this also include ".bin"?:
```suggestion
	supportedFileTypes = []string{".bin.gz", ".bin", ".json.gz", ".json"}
```

Reply:
I figured we'd skip `.bin` and just always gzip the binary snapshots - otherwise they'll be quite large since they include the raw memory (whereas the json compresses memory before encoding it in the json) and it just simplifies things.  Whereas we already have a mix of `.json` and `.json.gz` that we need to keep supporting.